harper-core/tests/text/linters/The Great Gatsby.snap.yml

```diff
@@ -1850,6 +1850,16 @@ Message: |
 
 
 
+Lint:    Miscellaneous (10 priority)
+Message: |
+    1407 | celebrated tenor had sung in Italian, and a notorious contralto had sung in
+    1408 | jazz, and between the numbers people were doing ‘‘stunts” all over the garden,
+         |                       ^~~~~~~ Use an apostrophe and `s` to form a noun’s possessive.
```

Comment:
This false positive will be hard to prevent. `numbers` is the last word in the prepositional phrase starting with `between` and `people` is the subject of the following clause. This would probably require a least a local parse tree!